Ical.Net.Tests/WikiSamples/RecurrenceWikiTests.cs

```diff
@@ -0,0 +1,451 @@
+﻿//
```

Comment:
We probably need a **naming convention** to link the Wiki test method to the actual Wiki content. Otherwise, maintenance or updates/additions might become hard.